libcxx/docs/ReleaseNotes/18.rst

```diff
@@ -109,6 +95,13 @@ LLVM 18
 LLVM 19
 ~~~~~~~
 
+- The ``LIBCXX_ENABLE_ASSERTIONS`` CMake variable that was used to enable the safe mode will be deprecated and setting
+  it will trigger an error; use the ``LIBCXX_HARDENING_MODE`` variable with the value ``safe`` instead. Similarly, the
+  ``_LIBCPP_ENABLE_ASSERTIONS`` macro will be deprecated (setting it to ``1`` still enables the safe mode in this
+  release while also issuing a deprecation warning). ``_LIBCPP_ENABLE_ASSERTIONS`` will be removed entirely in the next
```

Comment:
We don't issue a deprecation warning for `_LIBCPP_ENABLE_ASSERTIONS=1` in this release. Unless you meant LLVM 19 by "this", in which case I think the sentence is a bit unclear. I would suggest using explicit `LLVM <VERSION>` instead since that's unambiguous.